pkgs/top-level/python-packages.nix

```diff
@@ -16721,12 +16703,12 @@ in {
   update_checker = callPackage ../development/python-modules/update_checker {};
 
   uritemplate = buildPythonPackage rec {
```

Comment:
This should best be moved to its own file as well.

Reply:
Maybe its done on staging?

Reply:
No, I just checked again.